### ISSUE TITLE
configure error: '/tmp/mercury_conf.h.13699': No such file or directory

### ISSUE THREAD

**Issue by dram** [Mercury-Language/mercury#80]
When trying to build Mercury under Termux (a Linux environment for Android), following error is encountered:

```
...
checking for C shell executable... no
cp: cannot create regular file '/tmp/mercury_conf.h.13699': No such file or directory
configure: creating ./config.status
...
cmp: /tmp/mercury_conf.h.13699: No such file or directory
...
```

I think this error is related to following code in `configure`, which use `/tmp` instead of `TMPDIR`:

```
conf_h_copy=/tmp/mercury_conf.h.$$
```

**Comment by juliensf** [comment]
Fixed in commit 881ff72.  The fix will be available in the next rotd.

**Comment by dram** [comment]
Thanks, fix confirmed.
